go/chat/server.go

```diff
@@ -3646,6 +3646,33 @@ func (h *Server) RefreshParticipants(ctx context.Context, convID chat1.Conversat
 	return nil
 }
 
+func (h *Server) GetParticipants(ctx context.Context, convID chat1.ConversationID) (participants []chat1.ConversationLocalParticipant, err error) {
+	ctx = globals.ChatCtx(ctx, h.G(), keybase1.TLFIdentifyBehavior_CHAT_CLI, nil, h.identNotifier)
+	defer h.Trace(ctx, func() error { return err }, "GetParticipants")()
+	uid, err := utils.AssertLoggedInUID(ctx, h.G())
+	if err != nil {
+		return nil, err
+	}
+	uids, err := h.G().ParticipantsSource.Get(ctx, uid, convID, types.InboxSourceDataSourceAll)
+	if err != nil {
+		return nil, err
+	}
+	kuids := make([]keybase1.UID, 0, len(uids))
+	participants = make([]chat1.ConversationLocalParticipant, 0, len(uids))
```

Comment:
can move this down to the participants loop

Reply:
the `participants = make(`? Full disclosure this is from https://github.com/keybase/client/blob/34008521277058b8a6b1bcea2a6b83f8f296e253/go/chat/participantsource.go#L174-L188, maybe it would be good to make this into a helper function or something?